addons/web/static/src/legacy/js/core/translation.js

```diff
@@ -68,7 +68,7 @@ var TranslationDataBase = Class.extend(/** @lends instance.TranslationDataBase#
         url += '/' + (cacheId ? cacheId : Date.now());
         const paramsGet = {};
         if (modules) {
-            paramsGet.modules = modules.join(',');
+            paramsGet.mods = modules.join(',');
```

Comment:
Was this a bug?

Reply:
A nasty one. It is "mods" in the python function's signature. "modules" was basically ignored without warning in master. Thanks to my change in `route_wrapper` it does emit the warning now so I fixed the bug.

Reply:
Discovering this kind of silent failures can be scary.  You sometimes realize something never worked the way it was intended to.

Reply:
note that the same change should probably be applied to the `localization_service.js` file: it is the new code that fetches the translations